### PR TITLE
Standardize default error message

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** Our dashboard shows some ugly error messages and I noticed we're not handling the default error using `boom`, which is having us send errors back from the API in different formats.

So I thought this might be a breaking change, but if anyone has scripted around this then they're already dealing with the other error format coming back.

Addresses [CUMULUS-1456 ](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1456)

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/api/app/routes.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ router.use('/ems', ensureAuthorized, ems);
 // eslint-disable-next-line no-unused-vars
 router.use((error, req, res, next) => {
   log.error(error);
-  return res.status(500).send({ error: error.message });
+  return res.boom.badRequest(error.message, error);
```
Comment (laurenfrederick) [inline packages/api/app/routes.js]: Using `badImplementation` for some reason with `boom` obscures your error message

Comment (flamingbear) [reply packages/api/app/routes.js]: BadImplementation is a 5xx error and those are hidden from the user. 
https://hapi.dev/family/boom/?v=8.0.1#boombadimplementationmessage-data---alias-internal
"All 500 errors hide your message from the end user. Your message is recorded in the server log."

Comment (laurenfrederick) [reply packages/api/app/routes.js]: Yep, which is why I used `badRequest`, a 400, instead.

Comment (flamingbear) [reply packages/api/app/routes.js]: ok, I just saw "for some reason" and thought I was helping. :)

**Pull request merged**